ocaml/xapi/helpers.ml

```diff
@@ -2015,6 +2015,18 @@ let with_temp_out_ch_of_temp_file ?mode prefix suffix f =
   let@ path, channel = with_temp_file ?mode prefix suffix in
   f (path, channel |> with_temp_out_ch)
 
+let make_external_host_verified_rpc ~__context ext_host_address ext_host_cert
+    xml =
+  let@ temp_file, temp_out_ch =
```

Comment:
How to get the file removed?

Reply:
It is auto removed by `with_temp_file`:
```
let with_temp_file ?mode prefix suffix f =
  let path, channel = Filename.open_temp_file ?mode prefix suffix in
  finally (fun () -> f (path, channel)) (fun () -> Unix.unlink path)
```

Reply:
Then the file will not exist when the function returned. Is the file not requires anymore after that really?

Reply:
Yes, had tested this `make_external_host_verified_rpc` with a pool login, succeeded.